OpenRA.Game/Graphics/SpriteRenderable.cs

```diff
@@ -24,17 +24,29 @@ public struct SpriteRenderable : IRenderable, IFinalizedRenderable
 		readonly int zOffset;
 		readonly PaletteReference palette;
 		readonly float scale;
+		readonly float3 tint;
+		readonly float intensity;
```

Comment:
What is this for? It doesn't seem to get used other than for creating new `SpriteRenderable`s that don't use it more either?

Reply:
Vestigial state from an earlier implementation that kept tint and intensity separated until later in the rendering pipeline (this would have allowed an exact recreation of TS's worse-quality lighting behaviour).